arrayAnalyze/arrayAnalyze.test.js

```diff
@@ -0,0 +1,10 @@
+const arrayAnalyze = require('./arrayAnalyze');
+
+const object = arrayAnalyze([1, 8, 3, 4, 2, 6]);
+
+test('test to find average, min, max and length', () => {
+  expect(object.average).toBe(4);
+  expect(object.min).toBe(1);
+  expect(object.max).toBe(8);
+  expect(object.length).toBe(6);
+});
```

Comment:
This is fine, but one suggestion: you are testing different methods, so to have better feedback from the tests, you have each in its own test:

```js
test('should get the average', () => {
  expect(object.average).toBe(4);
})
test('should get the min', () => {
  expect(object.min).toBe(1);
})
// ....
});
```

You can see the difference in the output for passing tests with the `--verbose` flag: `npm t -- --verbose`.

If there are failing tests, you will have one test failing for the specific method.